tests/receivers/postgresql/bundled_test.go

```diff
@@ -38,6 +38,41 @@ import (
 	"github.com/signalfx/splunk-otel-collector/tests/testutils/kubeutils/manifests"
 )
 
+func TestPostgresqlDockerObserver(t *testing.T) {
+	testutils.SkipIfNotContainerTest(t)
+	if runtime.GOOS == "darwin" {
+		t.Skip("unable to share sockets between mac and d4m vm: https://github.com/docker/for-mac/issues/483#issuecomment-758836836")
+	}
+
+	testutils.AssertAllMetricsReceived(t, "bundled.yaml", "otlp_exporter.yaml",
+		postgresqldb, []testutils.CollectorBuilder{
+			func(c testutils.Collector) testutils.Collector {
+				cc := c.(*testutils.CollectorContainer)
+				cc.Container = cc.Container.WithBinds("/var/run/docker.sock:/var/run/docker.sock:ro")
+				cc.Container = cc.Container.WillWaitForLogs("Discovering for next")
+				cc.Container = cc.Container.WithUser(fmt.Sprintf("999:%d", testutils.GetDockerGID(t)))
+				return cc
+			},
+			func(collector testutils.Collector) testutils.Collector {
+				return collector.WithEnv(map[string]string{
+					"SPLUNK_DISCOVERY_DURATION": "10s",
+					// confirm that debug logging doesn't affect runtime
+					"SPLUNK_DISCOVERY_LOG_LEVEL": "debug",
+				}).WithArgs(
+					"--discovery",
+					"--set", "splunk.discovery.receivers.postgresql.config.username=otelu",
+					"--set", "splunk.discovery.receivers.postgresql.config.password=otelp",
+					"--set", "splunk.discovery.receivers.postgresql.config.tls::insecure=true",
+					//Disabling postgresql.backends metric that doesn't dependably show up in metrics during testing
+					"--set", "splunk.discovery.receivers.postgresql.config.metrics::postgresql.backends::enabled=false",
```

Comment:
This was strange. During testing I'd have a 50/50 shot of this `postgresql.backends` metric showing up or not when pulling metrics for comparison. 
Tried longer startup times and different methods of generating connections to try to get this metric to consistently show up but couldn't find any real rhyme or reason.

Open to ideas if it is imperative we have this enabled for testing.